transactions/add-access-nodes/2024/sept-3/README.md

```diff
@@ -0,0 +1,19 @@
+# Increase Access node Slot Limit
+
+> Sept 3rd, 2024 
+
+Increases the slot limit for Access nodes by TBD
```

Comment:
@joshuahannan - this [transaction](https://github.com/onflow/service-account/blob/main/templates/add_access_limits.cdc#L27) calls the  [setSlotLimits](https://github.com/onflow/flow-core-contracts/blob/cadence-0.42/contracts/FlowIDTableStaking.cdc#L902).

Should it instead be calling [setNodeOpenSlots](https://github.com/onflow/flow-core-contracts/blob/cadence-0.42/contracts/FlowIDTableStaking.cdc#L919) instead?

(This bump is slots is temporary and slot count should go back to 5 in the next epoch)

Reply:
This is the correct transaction, so this should be fine!

Reply:
thank you